pkgs/applications/networking/cisco-packet-tracer/customRequireFile.nix

```diff
@@ -0,0 +1,9 @@
+{ requireFile }:
+
+{ filename
+, sha256
+}: requireFile {
```

Comment:
This abstraction no longer is necessary.
`url` value can be taken from `meta.homepage`.